### ISSUE TITLE
The pager buttons and input aria-label attribute values can be focused due to negative tab index value. JAWS does not read them to the user. 

### ISSUE THREAD

**Issue by boyandim** [telerik/kendo-ui-core#2825]
The pager buttons and input aria-label attribute values can be focused due to negative tab index value. In this case the JAWS does not read them to the user. 

Please refer to the http://dojo.telerik.com/eYaka/5 example for demonstration. 

### Expected/desired behavior

The buttons and input to be focus-able so JAWS can read them. 

### Environment

* **Kendo UI version:** 2017.1.118
* **jQuery version:** 1.12.3
* **Browser:** [all ]

**Comment by Alex-Bubblemaster** [comment]
Raising priority. Requested in 1432833. Workaround in [this forum thread](https://www.telerik.com/forums/changing-tabindex-for-grid-pager-elements#VuHPNEk4nUqD6xpgchKf7Q)

**Comment by IvanDanchev** [comment]
Priority raised to 4. Pager buttons accessibility requested in 1444674.

**Comment by IvanDanchev** [comment]
Screen readers do not announce the pager input label. The item announced as selected in the dropdown (enabled through the **pageable.pageSizes** option) should also be associated with a "items per page" label (1451197).

**Comment by michal-ayash** [comment]
According to [MAS 1.3.1 - Info and Relationships](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7d), there is an accessibility issue with the grid page info:

Actual Result:

While navigate on page number edit field, screen reader does not provide proper information. label is not associated with page number edit field and the incorrect information read for selected value for items per page.

Expected Result:

When we navigate on page number edit field screen reader should provide proper information like page number 1.

Also Screen readers should read the label and items per page with their edit field and the correct information read for selected value for items per page.

 See [https://www.telerik.com/account/support-tickets/view-ticket/1451197](url)

**Comment by Dimitar-Goshev** [comment]
Dropdown aria-label issue reported in 1480991.

**Comment by Alex-Bubblemaster** [comment]
Also inquired about in 1487413

**Comment by nelito987** [comment]
Reported also in ticket ID: 1500927

**Comment by dmanova** [comment]
Issue reported also in 1506410. Please ping user when issue is done.

**Comment by elbazitay** [comment]
Any estimation for a fix ?
Tracking this issue, please ping when done.
